cpp/tests/CMakeLists.txt

```diff
@@ -330,7 +330,7 @@ ConfigureTest(STRINGS_TEST
     strings/chars_types_tests.cpp
     strings/combine/concatenate_list_elements_tests.cpp
     strings/combine/concatenate_tests.cpp
-    strings/combine/join_strings_tests
+    strings/combine/join_strings_tests.cpp
```

Comment:
Sorry, is this change unrelated to the `dlpack` change?

Reply:
It was a CMake anti-pattern that sneaked into the code-base. https://github.com/rapidsai/cudf/pull/8270 also fixes it.